internal/api/challenges.go

```diff
@@ -109,15 +109,24 @@ func (c *Client) GetChallenge(ctx context.Context, name string) (*Challenge, err
 }
 
 type ListChallengesOptions struct {
-	Category string
+	Category []string
+	Status   []string
 }
 
 func (c *Client) ListChallenges(ctx context.Context, opts *ListChallengesOptions) ([]*Challenge, error) {
 	var challenges []*Challenge
 	query := url.Values{}
-	if opts.Category != "" {
-		query.Set("category", opts.Category)
+	if opts.Category != nil {
```

Comment:
It's possible to iterate over a `nil` slice in Go - the `if` check is redundant.